deployment.rb

```diff
@@ -106,6 +107,11 @@ def load_configuration()
     config['dashboard_db_writer'] ||= config['db_writer'] + config['dashboard_db_name']
     config['pegasus_db_reader']   ||= config['db_reader'] + config['pegasus_db_name']
     config['pegasus_db_writer']   ||= config['db_writer'] + config['pegasus_db_name']
+
+    # Set AWS SDK environment variables from provided config.
+    ENV['AWS_ACCESS_KEY_ID'] ||= config['aws_access_key'] || config['s3_access_key_id']
```

Comment:
Talked offline - we'll duplicate the s3 keys as aws keys, then remove the s3 keys later on to reduce confusion.